packages/www/pages/dashboard/billing/index.tsx

```diff
@@ -375,10 +377,11 @@ const Billing = () => {
               </Flex>
             </Heading>
           </Flex>
-          {!products[user.stripeProductId]?.order ? (
+          {!standardProducts.includes(products[user.stripeProductId]?.name) ? (
             <Text variant="neutral">
-              The Hacker plan is free of charge up to 1000 minutes per month and
-              limited to 10 concurrent viewers per account.
```

Comment:
Don't we want this text anymore?

Reply:
Not for hackers! Because they are going to see their upcoming invoice, only custom plan users will see a text